Gemfile

```diff
@@ -4,7 +4,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby "3.1.2"
 
 # Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
-gem "rails", "~> 7.0.2"
+gem "rails", "~> 7.0.6"
```

Comment:
What do you think of removing the minor number? This way, dependabot could update by itself.

Reply:
Nice idea.

You mean `'~> 7.0'`, right?

Reply:
With `'>= 7.0.6'` dependabot will continue having problems updating.

But without them, maybe it allows a machine with rails `7.0.1` to run for example.
![image](https://github.com/inloco/incognia-demo-api-ruby/assets/5134842/98a3c270-17ba-40a9-8f1b-1a12aa8f4a46)
 
I'll read about it.

Reply:
Done

> But without them, maybe it allows a machine with rails 7.0.1 to run for example.

I've made some tests, it's not true